### PR TITLE
Fix liver abuse

### PR DESCRIPTION
**Pull request by monster860**
:cl: monster860
tweak: If you have corazone reagents will now process normally, this should fix the infinite-chems-liver-failure-thingy bug
/:cl:

### CODE REVIEW

**Comment by Amelia0010** [comment]
lol

File: code/modules/mob/living/carbon/life.dm [hunk 1]
```diff
@@ -624,7 +624,7 @@ GLOBAL_LIST_INIT(ballmer_windows_me_msg, list("Yo man, what if, we like, uh, put
 		L.damage += d
 
 /mob/living/carbon/proc/liver_failure()
-	reagents.metabolize(src, can_overdose=FALSE, liverless = TRUE)
+	reagents.metabolize(src, can_overdose=has_trait(TRAIT_STABLEHEART), liverless = !has_trait(TRAIT_STABLEHEART)) // make it so reagents process normally if you have corazone
```
Comment (Altoids1) [inline code/modules/mob/living/carbon/life.dm]: Modularize me, daddy

**Review by Altoids1 — CHANGES_REQUESTED** [review]

**Comment by Ratvar** [comment]
why not try merging this on teegee first

**Comment by Rayisken** [comment]
Is this a straight negative to corazone for preventing liver failure? 

It is used in higher tier cryo setups and that should be taken into consideration as well.

**Comment by monster860** [comment]
not on tg because tg can suck a big fat dick

**Review by Altoids1 — APPROVED** [review]

**Pull request merged**